packages/components/src/autocomplete/types.ts

```diff
@@ -167,10 +167,10 @@ type RichTextFormat = {
 	type: string;
 };
 type RichTextFormatList = Array< RichTextFormat >;
-type RichTextValue = {
+export type RichTextValue = {
 	text: string;
-	formats?: Array< RichTextFormatList >;
-	replacements?: Array< RichTextFormat >;
+	formats: Array< RichTextFormatList >;
+	replacements: Array< RichTextFormat >;
```

Comment:
This is a slight change, but fomats and replacements seem to be required in the rich text package?

Reply:
Have you considered moving this type (and the ones that are necessary to compose it) to the `@wordpress/rich-text` package? That makes sense from an abstraction point of view - I'd expect the rich text types should be with the rich text package. The only reason why they're here originally would be because `@wordpress/rich-text` wasn't typed yet, while `@wordpress/components` was in the process of being typed.

I'm also asking because `@wordpress/rich-text` is a dependency of `@wordpress/components`, and not the other way around, and we definitely should avoid circular dependencies.

Reply:
moved!